tests/functional/logger/test_logger.py

```diff
@@ -219,13 +219,14 @@ def test_package_logger_stream(stdout):
     # GIVEN package logger "aws_lambda_powertools" is explicitly set with no params
     set_package_logger(stream=stdout)
```

Comment:
suggestion: let's move this to a separate test folder altogether, as it's the only not self-contained test for logging we have (for good reasons)